stellar/deploy-contract.js

```diff
@@ -89,8 +89,9 @@ async function getInitializeArgs(config, chain, contractName, wallet, options) {
         case 'example': {
             const gatewayAddress = nativeToScVal(Address.fromString(chain?.contracts?.axelar_gateway?.address), { type: 'address' });
             const gasServiceAddress = nativeToScVal(Address.fromString(chain?.contracts?.axelar_gas_service?.address), { type: 'address' });
+            const itsAddress = nativeToScVal(chain?.contracts?.InterchainTokenService?.address, { type: 'string' });
```

Comment:
```suggestion
            const itsAddress = nativeToScVal(chain?.contracts?.InterchainTokenService?.address, { type: 'address' });
```
missed the `address` type in the suggestion